CompfyuiOllama.py

```diff
@@ -61,7 +62,7 @@ def ollama_vision(self, images, query, debug, url, model):
 
 """)
 
-        response = client.generate(model=model, prompt=query, images=images_b64)
+        response = client.generate(model=model, prompt=query, images=images_b64, keep_alive=str(-keep_alive) + "m")
```

Comment:
why is str conversion is with negative value: -keep_alive ?

Reply:
@stavsap You're right that doesn't make sense, I read that somewhere on reddit I think.  (I fixed it)